packages/wonder-blocks-form/src/components/text-area.tsx

```diff
@@ -313,15 +313,17 @@ const TextArea = React.forwardRef<HTMLTextAreaElement, TextAreaProps>(
     },
 );
 
-const VERTICAL_SPACING_PX = 10;
+const VERTICAL_SPACING = sizing.size_100;
 
 const styles = StyleSheet.create({
     textarea: {
         borderRadius: border.radius.radius_040,
         boxSizing: "border-box",
-        padding: `${VERTICAL_SPACING_PX}px ${spacing.medium_16}px`,
+        paddingBlock: VERTICAL_SPACING,
+        paddingInline: sizing.size_160,
         // This minHeight is equivalent to when the textarea has one row
-        minHeight: `calc(${VERTICAL_SPACING_PX * 2 + 2}px + ${font.lineHeight.medium})`,
+        minHeight: `calc(${VERTICAL_SPACING} * 2 + ${sizing.size_020} + ${font.lineHeight.medium})`,
+        ...focusStyles.focus,
     },
     default: {
         background: semanticColor.input.default.background,
```

Comment:
We continue to use the `semanticColor.input` tokens. In the next PR where I work on adding the TB theme, I'll see if we need to make any changes to the shared input tokens!